src/controllers/LocationController.js

```diff
@@ -0,0 +1,34 @@
+import dotenv from 'dotenv';
+import Customize from '../helpers/Customize';
+import { cities } from '../database/models';
+
+dotenv.config();
+/**
+ * @export
+ * @class LocationController
+ */
+class LocationController {
+  /**
+   * An admin should be able to add locations
+   * @static
+   * @param {object} req request object
+   * @param {object} res response object
+   * @memberof LocationController
+   * @returns {object} data
+   */
+  static async saveLocation(req, res) {
+    try {
+      const { name } = req.body;
+
+      const newLocation = await cities.create({
+        city: name
+      });
+
+
+      return Customize.successMessage(req, res, 'Location posted successfully', newLocation, 201);
+    } catch (err) {
+      return Customize.errorMessage(req, res, err.message, 500);
+    }
+  }
+}
+export default LocationController;
```

Comment:
@hezronkimutai  Thanks a lot to create this for us to create  a new city without the need for seeding data